### DISCUSSION TITLE
Linux desktop app?

### DISCUSSION THREAD

**Discussion by sabaini** [modelcontextprotocol/.github#24] [General]
I'd like to try MCP but I'm running Linux (Ubuntu 24.04) and as far as I can tell there's no desktop app for Linux. Are there plans for one? Or is there a way to connect MCP to the web app?

**Comment by theUpsider** [comment]
Was waking up, really hyped to try this out just to find that linux is not supported. Im a bit disappointed to be honest.

**Comment by dsp-ant** [comment]
I understand the disappointment. There is no Linux app at the moment. Note that Cody and Zed both support parts of MCP as well. They both run on Linux.

**Comment by ChristianHinge** [comment]
I ended up using https://github.com/chrishayuk/mcp-cli - it does currently not support anthropic models, but it works with ollama and OpenAI. I am also looking forward to a desktop implementation on linux. Arch linux has one, but I cannot get it to work with mcp
https://aur.archlinux.org/packages/claude-desktop-bin
